webcompat/webhooks/helpers.py

```diff
@@ -40,21 +27,21 @@ def parse_and_set_label(body, issue_number):
         # ('browser', 'firefox')
         browser = match_list.groups(0)[1].lower()
         dash_browser = '-'.join(browser.split())
-        set_label('browser-' + dash_browser, issue_number)
+        return 'browser-{name}'.format(name=dash_browser)
+    else:
+        return None
 
 
-def set_label(label, issue_number):
+def set_labels(payload, issue_number):
     '''Do a GitHub POST request to set a label for the issue.'''
     # POST /repos/:owner/:repo/issues/:number/labels
     # ['Label1', 'Label2']
-    payload = [label]
-    api_post('labels', payload, issue_number)
-
-
-def dump_to_db(title, body, issue_number):
-    url = extract_url(body)
-    issue_db.add(WCIssue(issue_number, title, url, body))
-    issue_db.commit()
+    headers = {
+        'Authorization': 'token {0}'.format(app.config['OAUTH_TOKEN'])
+    }
+    uri = 'https://api.github.com/repos/{0}/{1}/labels'.format(
+        app.config['ISSUES_REPO_URI'], issue_number)
+    return requests.post(uri, data=json.dumps(payload), headers=headers)
```

Comment:
Yeah. That was the essence of my comment on https://github.com/webcompat/webcompat.com/issues/1228#issuecomment-300952788
with 

> So Basically we are doing the same thing with two different functions. We probably would gain rationalizing a bit. :)

I can certainly modify a bit more. so we use the only one. I was hesitating and I was wondering if there was an initial reason for insulation.